README.md

```diff
@@ -26,3 +26,7 @@ ESLint configuration options. Refer to [documentation for VSCode extension](http
 Q: How to enable linting of Typescript code?
 
 A: Make sure that eslint configuration in your project has `typescript-eslint` plugin configured. See https://github.com/typescript-eslint/typescript-eslint for more information.
+
+Q: How to use this in a Yarn 2 project?
+
+A: Install ESLint in the project, run `yarn dlx @yarnpkg/pnpify --sdk base` ([docs](https://yarnpkg.com/advanced/editor-sdks)) and set `settings.nodePath` to `.yarn/sdks`.
```

Comment:
This documentation is a bit terse. A lot of users have trouble understanding server settings is my experience. Can you add a minimal .sublime-project example?

Reply:
Happy to expand, but can you show a sample `.sublime-project` with an `LSP-eslint` config in it? I've tried various permutations of `settings.LSP-eslint.settings.nodePath` as strings/dicts to no avail, nothing of relevance in Google either.

Reply:
```js
{
	"folders":
	[
		{
			"path": "...",
		},
	],
	"settings": {
		"LSP": {
			"LSP-eslint": {
				"settings": {
					"nodePath": "..."
				}
			}
		}
	},
}
```

Reply:
Thanks! Would be nice to document this in LSP's docs. It mentions [per-project overrides](https://lsp.readthedocs.io/en/latest/#per-project-overrides), but, in the absence of an `LSP-something` example, I assumed `LSP-eslint` would be treated as a standalone package.